src/onegov/feriennet/collections/billing.py

```diff
@@ -291,11 +292,11 @@ def __init__(self, session, period):
 
             WHERE "group" != 'manual'
         """)
-        self.billed_attendees = {
-            (r.username, r.group) for r in session.execute(
+        self.billed_attendees = [
+            r.attendee_id for r in session.execute(
                 select(stmt.c).where(stmt.c.period_id == period.id)
             )
-        }
+        ]
```

Comment:
Why did you switch from a set comprehension to a list comprehension? Did you misread it as a dict comprehension?

A set lookup should be faster, plus it gets rid of duplicate entries.